e2e/client.e2e.test.ts

```diff
@@ -402,6 +402,20 @@ describe('createVariable', () => {
   });
 });
 
+describe('listTags', () => {
+  it('should lsit the tags', async () => {
+    const w = await client.Tags.list(organizationName);
+    const item = w.items[0];
+    expect(item).toHaveProperty('name')
+  });
+});
+
+describe('destroyTag',  () => {
+    it('should delete a tag', async () => {
+        await client.Tags.delete('jwalab', {type: "tags", id: "tag-xPfX5s5839Sjfghj"});
```

Comment:
from my understanding the second param should be an array : `[{type: "tags", id: "tag-xPfX5s5839Sjfghj"}]`